rancher/rancher-ui/applications/stacks/adding-balancers/index.md

```diff
@@ -57,6 +57,12 @@ In our example, any traffic directed to port `80` on the host of the load balanc
 
 You will have the option to no longer publish source ports to the host by setting the access level of each individual “Listening Port” to `Internal`.  All internal ports can only be accessed by services within the same environment.
 
+### SSL Termination
+
+To add SSL termination to your load balancer, you can select the **SSL** checkbox. When you enable the checkbox, the **SSL Termination** tab will provide the ability to add certificates. In the **Certificate** dropdown, you can select the main certificate for the load balancers. 
+
+SSL Termination applies only to listeners with the `https` or `ssl` protocol selected. Additional Certificates will be used by clients that support SNI. Clients that do not support SNI will always use the main Certificate.
```

Comment:
The UI and API changed since when you wrote this.. instead of choosing ssl/https protocols for the listener there's a checkbox fr SSL in the UI.

This corresponds to a `io.rancher.loadbalancer.ssl.ports=443,8443` label, but we want to change all the loadbalancer settings from labels to entries in `rancher-metadata.json` so I would just not document SSL termination with compose for this week.